### DISCUSSION TITLE
Problem increasing apk size

### DISCUSSION THREAD

**Discussion by ghost** [ReVanced/revanced-discussions#973] [Question] [answered]
![IMG_20230423_001630](https://user-images.githubusercontent.com/28665018/233797999-a08dcbb1-7ca5-4f5e-bc0f-d8502c4d4e00.jpg)

Hi Admin

I found 1 problem about apk size

Take a look at the image and notice the files in the red box

Those are encrypted files that only work with the original apk unaltered and it is resource encryption 

After you change the files those files are useless it just increases the size of the apk when changed.

Hope you will delete it in the future, thanks.

Thank you for the great tool, wish you all the best.

**Comment by oSumAtrIX** [comment, marked as answer]
Tried deleting them?

**Reply by ghost** [reply]
it will disappear in the new tool update, right
